cfgov/v1/atomic_elements/atoms.py

```diff
@@ -43,6 +43,7 @@ class Hyperlink(blocks.StructBlock):
         '"Learn more" that is not descriptive on its own).',
     )
     url = blocks.CharBlock(default="/", required=False)
+    make_link_boldface = blocks.BooleanBlock(default=False, required=False)
```

Comment:
Similar to the other boolean naming comment, you could consider naming this `is_link_boldface`, which is both shorter and more in line with convention.

Reply:
changed in https://github.com/cfpb/consumerfinance.gov/pull/8191/commits/374073de5799829c008bc8e4281eddfd4d57220a